include/cpr/util.h

```diff
@@ -30,6 +30,8 @@ std::vector<std::string> split(const std::string& to_split, char delimiter);
 std::string urlEncode(const std::string& s);
 std::string urlDecode(const std::string& s);
 void secureStringClear(std::string& s);
+bool isTrue(const std::string& s);
```

Comment:
Based on the implementation it only matches on `TRUE`. But HTTP is "usually" not case sensitive. Please don't compare it case sensitive and add a short comment here, when it function matches.